### PR TITLE
Floating dropdown max-width and ellipsis fix

### PR DESCRIPTION
**Pull request by jonathanadler**
# ✨ Pull Request

### 💁 Description
Currently max-width is 20vw, which doesn't work well for mobile.
This PR fixes this issue, and keeps ellipsis behaviour intact

...

### 💭 Motivation
https://wix.slack.com/archives/CJRU3U97A/p1608802553336300

...

### 👀  PR is ready for review 

- [x] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_558/

File: src/components/FloatingDropdown/FloatingDropdownBase.tsx [hunk 1]
```diff
@@ -55,11 +53,13 @@ export class FloatingDropdownBase extends React.Component<
         <div className={classes.content}>
           <div className={classes.textContent}>
             <span className={classes.label}>{label}</span>
-            <div
-              className={classes.selectedValue}
-              data-hook={DATA_HOOKS.baseSelectedValue}
-            >
-              {selectedOption?.value || placeholder}
+            <div className={classes.selectedValueContainer}>
```
Comment (nitayneeman) [inline src/components/FloatingDropdown/FloatingDropdownBase.tsx:56]: I guess we didn't manage to avoid this wrapper... 😬

**Review by nitayneeman — APPROVED** [review]
Maybe worth adding a visual for the ellipsis?

File: src/components/FloatingDropdown/FloatingDropdownBase.tsx (hunk 1 above)
Comment (jonathanadler) [reply src/components/FloatingDropdown/FloatingDropdownBase.tsx:56]: 🤔  you know what, maybe we can.
I originally took this approach from here: https://css-tricks.com/flexbox-truncated-text/
But let me check, maybe I can get rid of it

**Comment by jonathanadler** [comment]
> Maybe worth adding a visual for the ellipsis?

🤦‍♂️  doh.

**Comment by jonathanadler** [comment]
@nitayneeman Added visual test and removed wrapper div

**Comment by kranux** [comment]
Hey, @jonathanadler any chances that diff here https://bo.wix.com/sheshesh-web?reportDir=search-results-widget-[unknown]-36d51f8ceb030c20ebd37210cdb1ebfe-f27f32b9-dcd8-4344-ae21-7e723680f576 is related with this fix?

I guess this is why we had `max-width` :)

**Comment by jonathanadler** [comment]
> Hey, @jonathanadler any chances that diff here [https://bo.wix.com/sheshesh-web?reportDir=search-results-widget-[unknown]-36d51f8ceb030c20ebd37210cdb1ebfe-f27f32b9-dcd8-4344-ae21-7e723680f576](https://bo.wix.com/sheshesh-web?reportDir=search-results-widget-%5Bunknown%5D-36d51f8ceb030c20ebd37210cdb1ebfe-f27f32b9-dcd8-4344-ae21-7e723680f576) is related with this fix?
> 
> I guess this is why we had `max-width` :)

It could be related to this, but we had other changes in the component that were released.
It doesn't seem to be related specifically to the max-width, since the max-width was targeting the content element, which in the test diff hasn't changed.
Can you send me the link to where you're using the floatingDropdown?
I would really prefer to try to fix it in your app, rather than revert our changes.
I don't mind creating a PR for you

**Comment by kranux** [comment]
Hi, 

this is an example site: https://andriusk9.wixsite.com/events/search-results/q-even/qc-events
Our source code is here: https://github.com/wix-private/site-search

**Comment by kranux** [comment]
For now I just freezed wix-ui-tpa to ~2.14.0

Let me know if I can help somehow by solving this in our code :)

**Comment by jonathanadler** [comment]
> For now I just freezed wix-ui-tpa to ~2.14.0
> 
> Let me know if I can help somehow by solving this in our code :)

Great, thanks!

**Pull request merged**